package.json

```diff
@@ -94,15 +94,16 @@
     "@nebula.gl/layers": "1.1.0-alpha.5",
     "@paypal/react-paypal-js": "^8.5.0",
     "@reatom/async": "^3.15.2",
-    "@reatom/core": "^3.8.3",
+    "@reatom/core": "^3.9.0",
     "@reatom/core-v2": "^3.1.4",
     "@reatom/hooks": "^3.5.5",
     "@reatom/logger": "^3.8.4",
     "@reatom/npm-react": "^3.8.10",
     "@reatom/react-v2": "^3.1.2",
-    "@sentry/react": "^7.113.0",
-    "@slack/web-api": "^7.0.4",
-    "@testing-library/react": "^14.2.1",
+    "@sentry/react": "^8.18.0",
+    "@slack/web-api": "^7.3.1",
+    "@testing-library/dom": "^10.3.2",
+    "@testing-library/react": "^16.0.0",
```

Comment:
Isn't it better to put testing libraries in `dev` dependencies?

Reply:
Yes. Actually it's a good question to those, who added them initially in main deps)

Reply:
to do in #19349